howl/tasks/Task.lua

```diff
@@ -168,6 +168,18 @@ function Task:Run(context, ...)
 
 	if s then
 		context.env.logger:success("%s finished", self.name)
+		if self.description=="Minify a file" then
+			--For minify tasks, show reduction percent
+			local oldFile = io:open(self.options.input,"r")
+			local newFile = io:open(self.options.output,"r")
+			local oldSize = oldFile:seek("end")
+			local newSize = newFile:seek("end")
```

Comment:
I'm not sure CC supports seeking to the end - we read in the file's content in the task so that could be done then.